src/server.tsx

```diff
@@ -73,11 +81,19 @@ const startServer = async () => {
   server.app.use(router.allowedMethods());
   server.app.use(async (ctx: any, next: any) => {
     await next();
-    const { render, status } = await renderSite(ctx.request.url);
-    if (status) {
-      ctx.response.status = Number(status);
+    if (ctx.request.path === '/robots.txt') {
+      if (ctx.request.hostname.toLowerCase() === 'freeboardgame.org') {
+        ctx.response.body = OPEN_ROBOTS_TXT;
+      } else {
+        ctx.response.body = RESTRICTIVE_ROBOTS_TXT;
+      }
```

Comment:
can you move this logic of rendering the robot txt to its own method? I think it is a little bit too much here in startServer

Reply:
I agree, changed